testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/AbstractSamlTest.java

```diff
@@ -73,6 +74,7 @@ public abstract class AbstractSamlTest extends AbstractAuthTest {
 
     static {
         try {
+            CryptoIntegration.init(Thread.currentThread().getContextClassLoader());
```

Comment:
In case you are wondering: This was necessary to run `SamlReverseProxyTest` isolated from the CLI, as then the crypto integration wasn't initialized yet. This call doesn't do any harm if it isn't initialized yet.